frame/society/src/lib.rs

```diff
@@ -1216,11 +1273,22 @@ pub mod pallet {
 								// Ban the voucher from vouching again
 								<Vouching<T, I>>::insert(&voucher, VouchingStatus::Banned);
 							},
+							BidKind::Action(deposit, _) => {
```

Comment:
I think this needs to be extracted to be elsewhere - this code is currently unreachable because we do not add action bidders to suspended candidates (see L1651, this is probably also where this logic should be moved)

Reply:
Yep, will do.

Reply:
Done.